tests/test_feature.py

```diff
@@ -168,7 +169,10 @@ async def _test_feature(feat, query_mock):
         if feat.attribute_setter is None:
             return
 
-        expecting_call = feat.id not in internal_setters
+        # IotStrip makes calls via it's children
+        expecting_call = feat.id not in internal_setters and not isinstance(
+            dev, IotStrip
+        )
```

Comment:
This feels rather hacky, but maybe it's fine considering it's test code? Otherwise this looks good to me, but we may want to consider adding a master control for tapo strips in a separate PR to make it consistent.